CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * [BUGFIX] Catch context error in the s3 bucket client. #5240
 * [BUGFIX] Fix query frontend remote read empty body. #5257
 * [BUGFIX] Fix query frontend incorrect error response format at `SplitByQuery` middleware. #5260
+* [BUGFIX] Ruler: Validate if rule group can be safely converted back to rule group yaml from protobuf message #5264
```

Comment:
updated :bow:

Reply:
Is this a bug fix, or more like an enhancement? @emanlodovice

Reply:
I think this is a bug @yeya24 because it prevents rulers from loading the rules of the user. Though the ruler roles back to the previous working set of rules but that only works if the ruler don't restart. If we restart we won't be able to load any of the rules.